comictaggerlib/ui/settingswindow.ui

```diff
@@ -401,17 +391,182 @@
+            </property>
+            <layout class="QGridLayout" name="gridLayout_8" rowstretch="0,0,0,0" columnstretch="0,0,0,0">
+             <property name="sizeConstraint">
+              <enum>QLayout::SetDefaultConstraint</enum>
+             </property>
+             <property name="leftMargin">
+              <number>0</number>
+             </property>
+             <property name="topMargin">
+              <number>6</number>
+             </property>
+             <property name="spacing">
+              <number>0</number>
+             </property>
+             <item row="3" column="1" colspan="3">
+              <widget class="QCheckBox" name="cbxShortMetadataNames">
+               <property name="sizePolicy">
+                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
+                 <horstretch>0</horstretch>
+                 <verstretch>0</verstretch>
+                </sizepolicy>
+               </property>
+               <property name="toolTip">
+                <string>Use the short name for the metadata styles (CBI, CR, etc.)</string>
+               </property>
+               <property name="text">
+                <string>Use &quot;short&quot; names for metadata styles</string>
+               </property>
+              </widget>
+             </item>
+             <item row="1" column="1">
+              <widget class="QLabel" name="lblOverlayStyle">
+               <property name="toolTip">
+                <string>The operation to perform when overlaying new source information onto the current</string>
+               </property>
+               <property name="text">
+                <string>Overlay Operation:</string>
+               </property>
+               <property name="alignment">
+                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
+               </property>
+               <property name="margin">
+                <number>6</number>
+               </property>
+              </widget>
+             </item>
+             <item row="1" column="2">
+              <widget class="QComboBox" name="cbxOverlayStyle">
+               <property name="sizePolicy">
+                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
+                 <horstretch>0</horstretch>
+                 <verstretch>0</verstretch>
+                </sizepolicy>
+               </property>
+               <property name="maximumSize">
+                <size>
+                 <width>16777215</width>
+                 <height>16777215</height>
+                </size>
+               </property>
+               <property name="toolTip">
+                <string>Overlay - When the new source metadata has non-empty values, overwrite the current metadata
+
+Add Missing - Any values that are empty in the current metadata that are in the new source metadata, add those values
+
+Combine - Combine lists and use the new (non-empty) source metadata for all other values</string>
```

Comment:
I don't really like the wording here. What do you think of
 
> Overlay - Overlays new metadata on top of the original.
> (Series=batman, Issue=1, Tags=[batman,joker,robin])
> \+ (Series=Batman, Publisher=DC Comics, Tags=[mystery,action])
> = (Series=Batman, Issue=1, Publisher=DC Comics, Tags=[mystery,action])
> 
> Add Missing - Metadata missing from the original is added from the the new metadata.
> (Series=batman, Issue=1, Tags=[batman,joker,robin])
> \+ (Series=Superman, Issue=10, Publisher=DC Comics, Tags=[superman,lois lane,lex luthor])
> = (Series=batman, Issue=1, Publisher=DC Comics, Tags=[batman,joker,robin])
> 
> Combine - Same as Overlay except lists (tags, characters, genres, credits, etc...) are combined.
> (Series=batman, Issue=1, Tags=[batman,joker,robin])
> \+ (Series=Batman, Publisher=DC Comics, Tags=[mystery,action])
> = (Series=Batman, Issue=1, Publisher=DC Comics, Tags=[batman,joker,robin,mystery,action])

But now that's a lot more text....

How about making this a radio box so each item can have it's own description on the screen and we can make the example the tool tip?

Reply:
Examples should make things easier to understand.

I'm thinking of a text box (on the right as there is space) that changes to reflect the current chosen or, a tabbed text box for the options.

This is kind of related but I think it makes sense to have a choice for the read styles and the source data (with commandline always in `overlay` with clearing of lists etc. and having primacy over read styles). So having two combo boxes makes the most sense then rather than radio.

Reply:
This is what I went with. Could possible have it switch tab when a combo box is changed.
![image](https://github.com/comictagger/comictagger/assets/1141189/359fc757-165d-4038-8ea0-6c6b59b4cf99)

Reply:
Read style and source info need to have similar text as they are the same operation but the metadata comes from a different place

Reply:
What about covering both? Something like:
```
Source Info: Overlays the new 'source info'  metadata on top of the current metadata as taken from the file name parser or read style (unless 'Overwrite metadata was selected)
Read Style: Overlays subsequent metadata after the first read style from the next read style as set by the user
```
The read style text above probably needs some simplification but that general idea?

Reply:
So we have the origin of the data "read", "source" and then we have "style" indicating there is some sort of style to the reading, then we have "info" which doesn't contribute any real meaning and doesn't follow the other control. They need to have consistent names. Imagine if on the front page we had this
<img width="256" alt="Screenshot 2024-03-17 at 4 19 18 PM" src="https://github.com/comictagger/comictagger/assets/7547075/761308de-b37a-485b-aa1e-70c433639a03">
That just adds confusion, because instead of keeping consistent names for the controls it uses completely different terms. I really care what terminology you use but it needs to be consistent. And if the name can indicate that "source" is from comicvine (or one of the others) then it needs a description as well (either as a tooltip or a label)

Reply:
IIRC you said to use "style" for the read and modify when it's user facing so I did that. "Data Source" IIRC I changed because of the space it takes up in the tagger window. And then in the settings it's still "Metadata Source", so I think that's the root of the problem I made. In the auto-tag window the option is labelled `Overwrite metadata` so maybe removing the word "Source" from the tagger window and simply put "Metadata" is the answer?

The tool tip for both the label and combox for source info is:
```
The operation to perform when overlaying source data (Comic Vine, Metron, GCD, etc.)
```
and for read style(s):
```
The operation to perform when overlaying read styles (ComicRack, ComicBookInfo, etc.)
```
Are they okay as is?

And the general format of using the one tab to explain both "Metadata" and "Read Style(s)" is okay?

Reply:
You're missing the point. That was just an example. I don't really care what terminology you use, it just needs to be consistent with the control it is next to.

Reply:
I've changed it to "Data Source" to match the tagger window.

I've changed the description to this:
```
Read Style: Overlays all the none empty values of the read style metadata on top of the current metadata (i.e. file name parsing, other read style(s)).
Data Source: Overlays all the none empty values from the data source (e.g. Comic Vine) on top of the current metadata (i.e. file name parsing, read style(s)).
```
I think now that with the wording there is no need for separate descriptions for "Read Style" and "Data Source"?